main.go

```
@@ -48,67 +54,133 @@ func (l *CompliancePlugin) Configure(req *proto.ConfigureRequest) (*proto.Config
 	}
 
-	stepActivities := append(activities, &proto.Activity{
-		Title:       "Collect data",
-		Description: "Collect data, and prepare collected data for validation in policy engine",
-		Steps:       collectSteps,
-	})
+	policyEvaluator := internal.NewPolicyEvaluator(ctx, l.logger, activities)
 
-	policyEvaluator := internal.NewPolicyEvaluator(ctx, l.logger, stepActivities)
+	// Simple use case for evaluating all policies against the data collected
+	evidences, err := policyEvaluator.Eval(ctx, data, request.PolicyPaths, l.policyData, l.config.PolicyLabels)
```

golangci-lint (2.12.2)

[error] avoid direct access to proto field request.PolicyPaths, use request.GetPolicyPaths() instead

(protogetter)